proknow-sdk-test/ScorecardTest/ScorecardTemplatesTest.cs

```diff
@@ -84,12 +142,32 @@ public async Task CreateAsyncTest()
         [TestMethod]
         public async Task DeleteAsyncTest()
         {
-            var testNumber = 2;
+            var testNumber = 3;
 
             // Create a scorecard template
             var computedMetrics = new List<ComputedMetric>();
             var customMetrics = new List<CustomMetric>();
-            var scorecardTemplateItem = await _proKnow.ScorecardTemplates.CreateAsync($"{_testClassName}-{testNumber}", computedMetrics, customMetrics);
+            var scorecardTemplateItem = await _proKnow.ScorecardTemplates.CreateAsync($"{_testClassName}-{testNumber}", computedMetrics, customMetrics, null);
```

Comment:
I think we can avoid sending null, since the parameter takes it as null by default
```suggestion
            var scorecardTemplateItem = await _proKnow.ScorecardTemplates.CreateAsync($"{_testClassName}-{testNumber}", computedMetrics, customMetrics);
```

Reply:
Removed